kustomize/monitoring/base/alertmanager-config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+data:
+  alertmanager.yml: |
+    route:
+        receiver: 'null'
+    receivers:
+        - name: 'null'
+kind: ConfigMap
+metadata:
+  name: dkube-alertmanager-config
+  namespace: dkube
```

Comment:
Do we really need to do it as configmap ? alertmanager.yml file needs to come from storage because it is being modified in d3 api controller and needs to persist. if we make alertmanager config as configmap, any changes written to the file location won't persist. another issue is that d3api is not using /etc/ location for alertmanager.yml